torch/csrc/jit/codegen/cuda/expr_evaluator.h

```diff
@@ -43,15 +43,15 @@ class TORCH_CUDA_API EvaluationContext {
 // Evaluates expressions in a Fusion IR, using the passed in
 // context (EvaluationContext) to query for concrete_values. The
 // evaluation context may override concrete values in the IR as well.
-class TORCH_CUDA_API ExpressionEvaluator : private IterVisitor {
+class TORCH_CUDA_API ExpressionEvaluator : protected IterVisitor {
```

Comment:
Noticed that we changed everything from `private` to `protected` but we don't have a new class that inherits from `ExpressionEvaluator`.